Could not read ZoneInfoMap. You are probably using Proguard wrong.

Hi,

This is the stacktrace without any proguard rules
```
Caused by: java.lang.RuntimeException: Could not read ZoneInfoMap. You are probably using Proguard wrong.
                                                       at net.danlew.android.joda.a.a(JodaTimeAndroid.java:42)
```

```
Failed to retrieve identifier: type=class net.danlew.android.joda.b$a name=joda_zoneinfomap
                                                    java.lang.NoSuchFieldException: joda_zoneinfomap
                                                        at java.lang.Class.getField(Class.java:891)
                                                        at net.danlew.android.joda.c.a(ResUtils.java:105)
                                                        at net.danlew.android.joda.d.a(ResourceZoneInfoProvider.java:113)
                                                        at net.danlew.android.joda.d.<init>(ResourceZoneInfoProvider.java:38)
                                                        at net.danlew.android.joda.a.a(JodaTimeAndroid.java:39)
```

Using these rules solve the problem but I wonder what do you recommend
```
-dontwarn org.joda.convert.**
-dontwarn org.joda.time.**
-keep class org.joda.time.** { *; }
-keep interface org.joda.time.** { *; }
-keep class net.danlew.** { *; }
```

--- a/library/proguard-rules.txt
+++ b/library/proguard-rules.txt
@@ -1,3 +1,6 @@
+# All the resources are retrieved via reflection, so we need to make sure we keep them
+-keep class net.danlew.android.joda.R$raw { *; }
+
 # These aren't necessary if including joda-convert, but
 # most people aren't, so it's helpful to include it.
 -dontwarn org.joda.convert.FromString

